src/slic3r/GUI/BBLTopbar.cpp

```diff
@@ -94,8 +94,8 @@ void BBLTopbarArt::DrawButton(wxDC& dc, wxWindow* wnd, const wxAuiToolBarItem& i
     int textX = 0, textY = 0;
 
     const wxBitmap& bmp = item.GetState() & wxAUI_BUTTON_STATE_DISABLED
-        ? item.GetDisabledBitmap()
-        : item.GetBitmap();
+        ? item.GetDisabledBitmapFor(wnd)
```

Comment:
I assume this one is likely due to using GTK3 and wx3.2, not tested with GTK2 and wx3.0.